lib/fog/libvirt/compute.rb

```diff
@@ -57,12 +57,16 @@ def initialize(options={})
           require 'libvirt'
         end
 
-        private
-
         def client
-          return @client if defined?(@client)
+          @client ||= Class.new do
+            def libversion
+              1002009
```

Comment:
Dunno but i feel this should be done inside tests only so it wont change the behaviour of a real execution.
Maybe you can push this part of the code to a initializer that runs before a specific tests for libvirt `1.2.9`?